src/views/datasets/fish/summary.vue

```diff
@@ -56,20 +56,30 @@
     <template #subtitle
       ><p>
         The data available on this site has been compiled from the
-        <a href="https://web.uri.edu/fishtrawl/"
+        <a
+          href="https://web.uri.edu/gso/research/fish-trawl/"
+          target="_blank"
+          rel="noreferrer noopener"
           >University of Rhode Island Graduate School of Oceanography Fish Trawl
           Survey</a
         >.
       </p>
       <p>
         <strong>To cite this data</strong>:
-        <a href="https://web.uri.edu/fishtrawl/data/">Fish Trawl Data Page</a>
+        <a
+          href="https://web.uri.edu/gso/research/fish-trawl/data/"
```

Comment:
Updated these links to reflect changes at `web.uri.edu`

Reply:
thanks!